src/openrct2/interface/Fonts.cpp

```diff
@@ -25,70 +25,42 @@
 #include "../localisation/language.h"
 
 #ifndef NO_TTF
-static TTFFontSetDescriptor TTFFontMSGothic = { {
+TTFFontSetDescriptor TTFFontMSGothic = { {
```

Comment:
Can you make them `const`?

Reply:
Hmm, I'm not sure. I can see why that'd be a desirable properly, but when I add the `const` keyword to these `TTFontSetDescriptor`s, clang ultimately complains that `gCurrentTTFFontSet` is not const — and I don't think it should be, as the font can change in-game…

Reply:
No, because these descriptors have the actual `TTF_Font` assigned into them:
```
/home/aaron/Repositories/OpenRCT2/src/openrct2/drawing/ttf.c: In function ‘ttf_initialise’:
/home/aaron/Repositories/OpenRCT2/src/openrct2/drawing/ttf.c:84:28: error: assignment of member ‘font’ in read-only object
             fontDesc->font = ttf_open_font(fontPath, fontDesc->ptSize);
                            ^
/home/aaron/Repositories/OpenRCT2/src/openrct2/drawing/ttf.c: In function ‘ttf_dispose’:
/home/aaron/Repositories/OpenRCT2/src/openrct2/drawing/ttf.c:106:32: error: assignment of member ‘font’ in read-only object
                 fontDesc->font = NULL;
                                ^
```